diff --git a/src/configure b/src/configure
index cefdbd2..2271f5c 100755
--- a/src/configure
+++ b/src/configure
@@ -75,6 +75,9 @@ elif [  "$os_VENDOR" == "Fedora" ]; then
     build_breakpad
 
     print_title "Run: qmake-qt5 && make "
+elif [  "$os_VENDOR" == "Gentoo" ]; then
+    build_breakpad
+    print_title "Run: qmake && make"
 
 elif [[ $OSTYPE == darwin* ]]; then
     cd $DEPS_DIR
@@ -84,5 +87,5 @@ elif [[ $OSTYPE == darwin* ]]; then
     xcodebuild -sdk macosx -project src/client/mac/Breakpad.xcodeproj -configuration Release -target Breakpad install DSTROOT=$DEPS_DIR/../bin INSTALL_PATH=/Frameworks ARCHS=x86_64 ONLY_ACTIVE_ARCH=YES MACOSX_DEPLOYMENT_TARGET=10.10 GCC_VERSION=com.apple.compilers.llvm.clang.1_0 > /dev/null
     brew upgrade libssh2 || brew install libssh2
 else
-    echo "Your operation system is not supported!"
+    echo "Your operation system $os_VENDOR is not supported!"
 fi
